auth0/v3/authentication/passwordless.py

```diff
@@ -79,8 +79,51 @@ def sms(self, client_id, phone_number, client_secret=None):
             data=data
         )
 
+    def login(self,
+        client_id,
+        client_secret,
+        username,
+        realm,
+        otp,
+        audience=None,
+        scope='openid'):
+
+        """Login using email/sms + OTP.
+
+        Args:
+            client_id (str): Client Id of the application.
+            
+            client_secret (str): Client Secret of the application.
+
+            username (str): Email address or phone number.
+
+            realm (str): email or sms
+
+            otp (str): Code received in the email.
+
+            audience (str): API identifier of the API
+
+            scope (str, optional): Scope to use. Defaults to 'openid'.
+        """
+
+        return self.post(
+            '{}://{}/oauth/token'.format(self.protocol, self.domain),
+            data={
+                'grant_type': 'http://auth0.com/oauth/grant-type/passwordless/otp',
+                'client_id': client_id,
+                'client_secret': client_secret,
+                'username': username,
+                'realm': realm,
+                'otp': otp,
+                'audience': audience,
+                'scope': scope,
+            }
+        )
+
     def sms_login(self, client_id, phone_number, code, scope='openid'):
         """Login using phone number/verification code.
+        
+        Note: this method should be depcrecated in the future.
```

Comment:
Let's print a warning instead (watch for the import)

See reference https://github.com/auth0/auth0-python/blob/0f82f78e825cbd49957651cdead8b7f853f4af59/auth0/v3/authentication/users.py#L44